tools/wasme/cli/pkg/deploy/istio/istio_provider.go

```diff
@@ -463,6 +486,22 @@ func (p *Provider) makeIstioEnvoyFilter(filter *v1.FilterSpec, image pull.Image,
 	}, nil
 }
 
+// Returns true if istio version is 1.6.x or older
+func isOlderIstio(istioVersion string) bool {
+	parts := strings.Split(istioVersion, ".")
+
+	// check minor version
+	minor, err := strconv.Atoi(parts[1])
+	if err != nil {
+		logrus.WithField("istioVersion", istioVersion).Warn("unable to determine istio version, assuming 1.7+")
```

Comment:
might be helpful to include the error here as well

Reply:
Added!